IPs/RSKIP171.md

```diff
@@ -0,0 +1,52 @@
+# Title
+
+|RSKIP          |171           |
+| :------------ |:-------------|
+|**Title**      |Arbitrary-Length Data Return Mechanisim |
```

Comment:
I don't like this title. I'd rather it be clearer as to exactly what is being changed instead of pointing towards the general functionaly. ymmv

Reply:
something like "Clean EVM Internal State for Call-Like Opcodes"?

Reply:
renamed to "Clean EVM internal buffer in call-like opcodes"